nameserver/nameserver.go

```diff
@@ -176,6 +281,23 @@ func (n *Nameserver) deleteTombstones() {
 	n.entries.filter(func(e *Entry) bool {
 		return e.Tombstone == 0 || now-e.Tombstone <= int64(tombstoneTimeout/time.Second)
```

Comment:
Noup, because otherwise the `stopped` entries of e.g. destroyed containers won't get GC'd.